.gitmodules

```diff
@@ -6,7 +6,7 @@
 	url = https://github.com/surge-synthesizer/tuning-library.git
 [submodule "libs/surgesynthteam_tuningui"]
 	path = libs/surgesynthteam_tuningui
-	url = https://github.com/surge-synthesizer/surgesynthteam_tuningui.git
```

Comment:
Hi @FulopNandor , can you switch back the repo to surge-synthesizer ?

Reply:
Hi @asb2m10 and @baconpaul , I have switched back that,

Reply:
Great and I merged your change to that repo so you should update it to head in the submodules from surge

Reply:
Thank you very-very much! I am going to do that. Have a nice weekend!:-)